backend/src/test/java/gov/cdc/usds/simplereport/service/BaseServiceTest.java

```diff
@@ -52,13 +54,16 @@
   @Autowired protected HibernateQueryInterceptor _hibernateQueryInterceptor;
   @MockBean private CurrentOrganizationRolesContextHolder _currentOrganizationRolesContextHolder;
 
+  @Autowired private SupportedDiseaseRepository _supportedDiseaseRepo;
```

Comment:
formatting nit: should this be in the same block as the other declarations?

Reply:
Good catch! Updated